src/SDKs/SqlManagement/Management.Sql/Generated/SqlManagementClient.cs

```diff
@@ -442,19 +447,6 @@ public partial class SqlManagementClient : ServiceClient<SqlManagementClient>, I
         /// </summary>
         public virtual IServerVulnerabilityAssessmentsOperations ServerVulnerabilityAssessments { get; private set; }
 
-        /// <summary>
-        /// Initializes a new instance of the SqlManagementClient class.
-        /// </summary>
-        /// <param name='httpClient'>
-        /// HttpClient to be used
-        /// </param>
-        /// <param name='disposeHttpClient'>
-        /// True: will dispose the provided httpClient on calling SqlManagementClient.Dispose(). False: will not dispose provided httpClient</param>
-        protected SqlManagementClient(HttpClient httpClient, bool disposeHttpClient) : base(httpClient, disposeHttpClient)
```

Comment:
These two constructors were automatically removed. I'm not sure what triggered the change.